doc/format.rst

```diff
@@ -717,7 +726,9 @@ Other points to keep in mind
   (i.e. scalar types, sequence types), those arguments can be documented
   with type `array_like`.
 
-* Links : If you need to include hyperlinks in your docstring, note that
+* Links : Sphinx will automatically create hyperlinks to module, function,
```

Comment:
Is this recommended practice? I think most libraries use `:pyfunc:` etc. Genuinely asking, I don't know.

Reply:
It's what is done in SciPy, I think. I think Sphinx only needs more information if there is a name collision.

Reply:
It's fine with me to add something about this being a project specific thing and mention the more explicit options, but if it is very common for projects to use single backticks, I think something should be mentioned. What can we say without over-stepping numpydoc's role?

Reply:
Let's add the common footnote about default role.